### PR TITLE
 [issue #2][UI] Banner hapolearn

### PR DESCRIPTION
**Pull request by hapo-tungdq**
#2 add EOF

### CODE REVIEW

**Comment by hapo-hoand** [comment]
LGTM

**Comment by hapo-hoand** [comment]
LGTM

**Comment by hapo-hoand** [comment]
LGTM

File: scss/_banner.scss [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  .background-hapo-learn-banner {
+    width: 100%;
+    height: 80%;
+    background-image: url(../assets/img/Hapo_Learn_banner_1.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .hapo-learn-title {
+    position: absolute;
```
Comment (hapo-nghialuu) [inline scss/_banner.scss:15]: dùng flexbox

**Comment by hapo-hoand** [comment]
LGTM

**Review by hapo-nghialuu — APPROVED** [review]

**Comment by hapo-nghialuu** [comment]
LGTM

**Pull request merged**